aio/deploy/helm-chart/kubernetes-dashboard/Chart.yaml

```diff
@@ -12,9 +12,9 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-apiVersion: v1
+apiVersion: v2
 name: kubernetes-dashboard
-version: 3.0.2
+version: 3.1.0
```

Comment:
I feel we should advertise the breaking change by bumping major version, don't you think?

Reply:
sure!

Reply:
done, pta
update the readme as well